c2corg_ui/static/js/documentediting.js

```diff
@@ -141,39 +141,9 @@ app.DocumentEditingController = function($scope, $element, $attrs,
   this.alerts = appAlerts;
 
   /**
-   * first next step would be 2 by default
-   * @export
-   */
-  this.next_step = 2;
-
-  /**
-   * Current step is 1 by default
-   * @export
-   */
-  this.current_step = 1;
-
-  /**
-   * Waypoint type initialised in the edit.html
-   * @export
-   */
-  this.waypoint_type;
-
-  /**
-   * Previous step is 0 by default
-   * @export
-   */
-  this.previous_step = 0;
-
-  /**
-   * Max possible steps for creation/edition
-   * @export
+   * @type {boolean}
+   * @public
```

Comment:
you will notice that nearly every @export has changed to @public since Alex created the outingediting.js that inherits from documentediting.js, I think there is a reason for that.